src/app/components/FormInput.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { InputProps, Input as RebassInput } from '@rebass/forms/styled-components';
+import { DefaultError, DefaultState, FormInput as ReactFormInput } from 'typed-react-form';
+import { FormInputProps } from 'typed-react-form/dist/elements/FormInput';
+
+export const FormInput = <
+  T,
+  Key extends keyof T,
+  Value extends T[Key] | T[Key][keyof T[Key]],
+  State extends DefaultState = DefaultState,
+  Error extends string = DefaultError
+>({
+  form,
+  children,
+  ...rest
+}: Omit<Omit<InputProps, 'form'> & FormInputProps<T, State, Error, Key, Value>, 'as'>) => (
+  <RebassInput as={ReactFormInput} form={form as any} {...rest}>
```

Comment:
Can we prevent the `any` here?

Reply:
Possibly, but I don't know how. The `@rebass/forms` input has a prop `form: string;` but the `typed-react-form` `FormInput` expects a `Form` object. Do you have suggestions for how to handle this?

Reply:
What is it used for exactly?